receiver/mysqlreceiver/go.mod

```diff
@@ -13,6 +13,7 @@ require (
 )
 
 require (
+	github.com/DATA-DOG/go-sqlmock v1.5.0
```

Comment:
Is this actually used in the receiver?

Reply:
removed that and some other extraneous imports as well.